src/watcher.js

```diff
@@ -55,6 +55,10 @@ Watcher.prototype = {
             }
           });
         });
+
+        streams[bundleFile].on('error', function (error) {
+          that._events.emit('error', error, bundleFile);
+        });
```

Comment:
Should I use `that._events.emit.bind(that, error, bundleFile)` (and similarly below)?